rakefile.rb

```diff
@@ -95,6 +101,9 @@
 desc "Execute default tasks"
 task :default => [ :vars, :gitlink, :unit, :integ, :spec, :pack ]
 
+desc "Execute default tasks for .NET Core solution"
+task :dotnet => [ :default, :unit_dotnet, :pack_dotnet]

```

Comment:
@adamralph [suggested](https://github.com/FakeItEasy/FakeItEasy/issues/531#issuecomment-236426591) having `dotnet` run the whole shebang. I saw no harm.